src/quo2/components/colors/color_picker/style.cljs

```diff
@@ -0,0 +1,47 @@
+(ns quo2.components.colors.color-picker.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(def color-picker-container
+  {:max-width       338
```

Comment:
Please don't hard code container width
colors width, height 48, margin between colors 19, so wouldn't container width will be fixed?

Reply:
I'm using fixed container width & `:justify-content :space-between` to space the components. I thought it was better than using margin in this case. Any specific reason for avoiding this approach in this scenario?

Reply:
That's a really good question. I am not sure which is recommended way, let's confirm with the design team.

Reply:
@pedro-et, Whole color-picker container has width 335 (40 x 6+19 x 5), where 40 is color item's size and 19 is margin between them.
But lets say we have a wide android device. For that should we just hard code these margins, or spread these color items around as per available gap `(:justify-content :space-between)`?

Reply:
if its slightly larger just keep the side margins and space the elements evenly. If its significantly larger maybe just had an extra column. @Parveshdhull